nsis/src/nsis/NsisAnalyzer.java

```diff
@@ -139,4 +142,56 @@ private void resolveStrings(Instruction instr, MemoryBlock stringsBlock)
     }
   }
 
+  /**
+   * Resolve the control flow for the specified instruction
+   * 
+   * @param instr the instruction
+   * @param entriesBlock the memory block containing the instructions
+   * @param program
+   * @throws MemoryAccessException
+   */
+  private void resolveControlFlow(Instruction instr, MemoryBlock entriesBlock,
+      ReferenceManager referenceManager) throws MemoryAccessException {
+    String mnemonic = instr.getMnemonicString();
+    int instructionNumber;
+    switch (mnemonic) {
+      case "Call":
+        instr.setFlowOverride(FlowOverride.CALL);
+        instructionNumber = instr.getInt(NsisConstants.ARG1_OFFSET);
+        referenceManager.addMemoryReference(instr.getAddress(),
+            getInstructionAddress(entriesBlock, instructionNumber), RefType.UNCONDITIONAL_CALL,
+            SourceType.ANALYSIS, NsisConstants.ARG1_INDEX);
+        break;
+
+      case "Jmp":
+        instr.setFlowOverride(FlowOverride.BRANCH);
+        instructionNumber = instr.getInt(NsisConstants.ARG1_OFFSET);
+        referenceManager.addMemoryReference(instr.getAddress(),
+            getInstructionAddress(entriesBlock, instructionNumber), RefType.UNCONDITIONAL_JUMP,
+            SourceType.ANALYSIS, NsisConstants.ARG1_INDEX);
+        instr.setFallThrough(null);
+        break;
+
+      case "Return":
+        instr.setFlowOverride(FlowOverride.CALL_RETURN);
+        instr.setFallThrough(null);
+        break;
+
+      default:
+        break;
+    }
+  }
+
+  /**
+   * Get the address associated to the instruction number
```

Comment:
Could you please document `instructionNumber` in more detail, stating that the function will find the address in memory of the instruction on the position `instructionNumber` in the entries block?